source/includes/steps-fle-create-data-encryption-key.yaml

```diff
@@ -157,6 +181,39 @@ content: |
            npm package. See the `npmjs documentation on the uuid-base64
            package <https://www.npmjs.com/package/uuid-base64>`_ for
            installation instructions.
+     .. tab::
+        :tabid: python
+
+        .. code-block:: python
+           
+           from pymongo import MongoClient
+           from pymongo.encryption_options import AutoEncryptionOpts
+           import base64
+
+           connectionString = "mongodb://localhost:27017"
+           keyVaultNamespace = "encryption.keyvault"
+           fle_opts = AutoEncryptionOpts(
+               kms_providers,
+               key_vault_namespace,
+               mongocryptd_bypass_spawn = True 
+           )
+ 
+           client = MongoClient(
+               connection_string,
+               auto_encryption_opts = fle_opts 
+           )
+ 
+           client_encryption = pymongo.encryption.ClientEncryption(
```

Comment:
Would it make sense to also add an import:
```
from pymongo.encryption import ClientEncryption
```
and use that here to be consistent with `MongoClient`?